models/public/caffenet/model.yml

```diff
@@ -13,8 +13,7 @@
 # limitations under the License.
 
 description: >-
-  CaffeNet model. Used for classification. For details see paper
-  <https://arxiv.org/abs/1408.5093>.
+  CaffeNet* model is used for classification. For details see paper <https://arxiv.org/abs/1408.5093>.
```

Comment:
I don't think these asterisks make sense here, since there is no footnote to explain them.

Reply:
These asterisks exist in documentation files and script just takes it as difference.

Reply:
This doesn't really address my comment... but I suppose it would be difficult to automatically distinguish between footnotes and asterisks used for other reasons, so never mind.